test/Microsoft.Framework.Logging.Test/LoggerFactoryExtensionsTest.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.Framework.Logging.Test
     public class LoggerFactoryExtensionsTest
     {
 #if ASPNET50
```

Comment:
The `project.json` does not have `aspnetcore50` so the test project is not supposed to build against core clr. Should I enable that in this PR  or should we track that as a separate item ? IMO we should be looking at it separately. The project depends on Logging.Serilog project which is not doesn't compile for `coreclr` .